api/api.go

```diff
@@ -108,8 +123,8 @@ func GetState(w http.ResponseWriter, r *http.Request, d *db.Database) {
 // GetStateActivity returns the activity (version history) of a State
 func GetStateActivity(w http.ResponseWriter, r *http.Request, d *db.Database) {
 	w.Header().Set("Access-Control-Allow-Origin", "*")
-	st := util.TrimBasePath(r, "api/state/activity/")
-	activity := d.GetStateActivity(st)
+	lineage := util.TrimBasePath(r, "api/state/activity/")
+	activity := d.GetStateActivity(lineage)
```

Comment:
Should this be renamed to `GetLineageActivity`, along with the calling function?

Reply:
Yes, it will be better I think